src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/PSSwaggerUtility/PSSwaggerUtility.psm1

```diff
@@ -50,7 +50,7 @@ function Get-RequiredModulesPath
         [System.Management.Automation.PSModuleInfo]
```

Comment:
Do not touch this file, please revert any changes you have.

Reply:
Sure